root/layout/components/ExternalLinks.js

```diff
@@ -75,12 +76,21 @@ const ExternalLinks = ({entity, empty, heading}) => {
           text={l('Official homepage')}
         />,
       );
+    } else if (/^blog$/.test(linkType.name)) {
```

Comment:
Would be nice to add a comment to allow finding such hard-coded database value.
```suggestion
    // Hard-coded condition on data: musicbrainz.link_type.name = 'blog'
    } else if (/^blog$/.test(linkType.name)) {
```
That can be discussed/done outside of this PR though.

Reply:
Or just compare against `linkType.gid` here? And add a `BLOG_LINK_TYPE` constant to root/static/scripts/common/constants.js to make it nicer.

Reply:
(Though that's a bit more annoying since you have to factor in the entity types, and it's unlikely the name should change, so a comment seems fine.)